x-pack/plugins/security_solution/public/timelines/components/timeline/tabs/query/index.tsx

```diff
@@ -239,13 +239,21 @@ export const QueryTabContentComponent: React.FC<Props> = ({
             },
           },
         });
+        telemetry.reportOpenNoteInExpandableFlyoutClicked({
+          location: timelineId,
+        });
+        telemetry.reportDetailsFlyoutOpened({
+          location: timelineId,
+          panel: 'right-left',
```

Comment:
I think the `panel` type is one of right, left and preview. not sure if this will go through

Reply:
yes you are correct, I had originally added that new `right-left` entry as an option but reverted the change and forgot to undo a few places I was using it... I decided to go with just `left`, while it does not completely represents what's happening I feel like it's the closest.
Thoughts?